spec/features/users/show_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+describe "User Profile Page", type: :feature do
+  let(:password) { 'password' }
+  let(:user) { FactoryBot.create(:user, password: password) }
+  let(:user2) { FactoryBot.create(:user, password: password, full_name: "#{user.full_name}_other") }
+
+  it "lists the user's full name" do
+    sign_in user
+    visit profiles_me_path
+
+    expect(page).to have_text(user.full_name)
+  end
+
+  it "lists the user's email" do
+    sign_in user
+    visit profiles_me_path
+
+    expect(page).to have_text(user.email)
+  end
+
+  it "lists the user's description" do
+    sign_in user
+    visit profiles_me_path
+
+    expect(page).to have_text(user.description)
+  end
+
```

Comment:
IMO there should be two tests here:

- User profile page lists the user's description (if it has one)
- User profile page doesn't list the user's description (if it has none)

Reply:
keeping our approach towards test coverage in mind, let's stick with the tests we have and not over-do it :)

Reply:
:+1: